utils/michigan.py

```diff
@@ -0,0 +1,530 @@
+import uuid
+
+import numpy as np
+import pandas as pd
+
+from utils.clean import StateCleaner
+from utils.constants import (
+    MI_CON_FILEPATH,
+    MI_CONTRIBUTION_COLUMNS,
+    MI_EXP_FILEPATH,
+    MI_EXPENDITURE_COLUMNS,
+    MICHIGAN_CONTRIBUTION_COLS_RENAME,
+    MICHIGAN_CONTRIBUTION_COLS_REORDER,
+)
+from utils.preprocess_mi_campaign_data import (
+    read_contribution_data,
+    read_expenditure_data,
+)
+
+
+class MichiganCleaner(StateCleaner):
+    entity_name_dictionary = {
+        "cfr_com_id": "original_com_id",
+        "f_name": "first_name",
+        "l_name_or_org": "last_name",
+        "employer": "company",
+        "doc_stmnt_year": "year",
+        "exp_desc": "purpose",
+        "transaction_type": "schedule_desc",
+    }  # map to entity types listed in the schema
+
+    # columns to be added {full_name, entity_type, state, party, company,
+    # transaction_id,
+    #  "donor_id", ""}
```

Comment:
In the clean.py, instead of creating an entity mapping dictionary property, I think we need to create a function in which we create that dictionary, I think we need to follow that structure in our state cleaner?

Reply:
When did we decide on that? My understanding was that the dictionary was just going to be initiated like this?